src/rating.css

```diff
@@ -0,0 +1,32 @@
+.product-card__rating {
+  display: flex;
+  align-items: center;
+  justify-content: space-between;
+  padding: 0 16px;
+}
```

Comment:
how you changed naming for button and rating same way change for rating as well in `rating.css`